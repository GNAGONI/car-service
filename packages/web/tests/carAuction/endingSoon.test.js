import React from 'react'
import renderer from 'react-test-renderer'
import EndingSoon from '../../components/carAuction/endingSoon'

it('renders correctly', () => {
  const tree = renderer.create(<EndingSoon />).toJSON()
  expect(tree).toMatchSnapshot()
})
