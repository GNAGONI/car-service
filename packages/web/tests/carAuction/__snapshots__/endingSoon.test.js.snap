// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="content auction-featured bg-before-default"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        
      </div>
    </div>
  </div>
  <div
    className="content-holder"
  >
    <div
      className="auction-slider"
    >
      <div
        className="slick-slider slick-initialized"
        dir="ltr"
      >
        <div
          className="btn-prev"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        />
        <div
          className="slick-list"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={null}
          onMouseMove={null}
          onMouseUp={[Function]}
          onTouchCancel={null}
          onTouchEnd={[Function]}
          onTouchMove={null}
          onTouchStart={[Function]}
          style={Object {}}
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "",
              }
            }
          >
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-3}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-2}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={-1}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active slick-current"
              data-index={0}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={1}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={false}
              className="slick-slide slick-active"
              data-index={2}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide"
              data-index={3}
              onClick={[Function]}
              style={
                Object {
                  "outline": "none",
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={4}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={5}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={6}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              aria-hidden={true}
              className="slick-slide slick-cloned"
              data-index={7}
              onClick={[Function]}
              style={
                Object {
                  "width": 0,
                }
              }
              tabIndex="-1"
            >
              <div>
                <div
                  className="post-auction  col-lg-12 col-md-12 col-sm-12"
                >
                  <div
                    className="holder"
                  >
                    <div
                      className="img-thumbnail"
                    >
                      <img
                        alt="Default car"
                        height="228"
                        src=""
                        width="380"
                      />
                    </div>
                    <div
                      className="frame"
                    >
                      <h3>
                        <a
                          href="/car-auctions/car-auction-details?id="
                          onClick={[Function]}
                        >
                          2006 Vauxhall Corsa 1.6 SRI
                        </a>
                      </h3>
                      <span
                        className="price"
                      >
                        £
                        160
                         
                        <span
                          className="text"
                        >
                          ( 
                           bids )
                        </span>
                      </span>
                      <time
                        className="time"
                      />
                      <div
                        className="btn-block extend"
                      >
                        <button
                          className="btn btn-primary"
                          disabled={false}
                          onClick={[Function]}
                          primary={true}
                          type="button"
                        >
                          View this auction
                        </button>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="btn-next"
          onClick={[Function]}
          style={
            Object {
              "display": "block",
            }
          }
        />
      </div>
    </div>
  </div>
</section>
`;
